docs/reference/02-find.md

```diff
@@ -0,0 +1,82 @@
+# Basic API
+
+Find, update and delete records.
+
+The examples use the following prisma schema:
+
+```prisma
+model Post {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    updatedAt DateTime @updatedAt
+    published Boolean
+    title     String
+    content   String?
+
+    comments Comment[]
+}
+
+model Comment {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    content   String
+
+    post   Post @relation(fields: [postID], references: [id])
+    postID String
+}
+```
+
+## Reading data
+
+### Find many records
+
+```go
+posts, err := client.Post.FindMany(
+    db.Post.Title.Equals("hi"),
+).Exec(ctx)
+```
+
+If no records are found, this returns an empty array without returning an error (like usual SQL queries).
+
+### Find one record
+
+```go
+post, err := client.Post.FindOne(
+    db.Post.ID.Equals("123"),
+).Exec(ctx)
+
+if err == db.ErrNotFound {
+    log.Printf("no record with id 123")
+}
+```
+
+This returns an error of type `ErrNotFound` (exported in the `db` package) if there was no such record.
```

Comment:
returns an `ErrNotFound` type (exported **by** the `db` package).

Reply:
I will do

> This returns an `ErrNotFound` error (exported by the generated client) if there was no such record.